### DISCUSSION TITLE
Home Assistant Version

### DISCUSSION THREAD

**Discussion by bmos** [andrew-codechimp/bn-device-test#62] [General]
Just dropping this here sine it's convenient, although obviously outside the scope of Actions testing.

The HA version in requirements.txt and hacs.json are not the same.
You updated the HA version in requirements [here](https://github.com/andrew-codechimp/HA-Battery-Notes/commit/072615187a351c08c3a245491d347635d3d42e97#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552) recently but did not in the hacs.json.

**Comment by andrew-codechimp** [comment]
I updated it there so I could test the later HA version which supports translated services.  It will still work with the older version, just not pick up the translations.
I am wondering how long should I support old versions given most people seem to update HA fairly frequently.
